src/nvim/ex_cmds2.c

```diff
@@ -1887,45 +1887,6 @@ typedef struct {
   const linenr_T final_lnum;
 } GetBufferLineCookie;
 
-/// Get one line from the current selection in the buffer.
-/// Called by do_cmdline() when it's called from cmd_source_buffer().
-///
-/// @return pointer to allocated line, or NULL for end-of-file or
-///         some error.
-static char_u *get_buffer_line(int c, void *cookie, int indent, bool do_concat)
-{
-  GetBufferLineCookie *p = cookie;
-  if (p->curr_lnum > p->final_lnum) {
-    return NULL;
-  }
-  garray_T ga;
-  ga_init(&ga, sizeof(char_u), 400);
-  ga_concat(&ga, ml_get(p->curr_lnum++));
-  if (do_concat && vim_strchr(p_cpo, CPO_CONCAT) == NULL) {
-    CONCAT_CONTINUED_LINES(&ga, p->curr_lnum <= p->final_lnum,
-                           ml_get(p->curr_lnum), p->curr_lnum++);
```

Comment:
`get_buffer_line` isn't needed anymore, which means `CONCAT_CONTINUED_LINES` is only used in one place now.

I could get rid of it and inline it in within `getsourceline`, which is its only usage now?